pkg/influx/datasource.go

```diff
@@ -26,9 +27,14 @@ type InfluxRunner struct {
 func (r *InfluxRunner) runQuery(ctx context.Context, q string) (*influxdb2.QueryTableResult, error) {
 	return r.client.QueryApi(r.org).Query(ctx, q)
 }
+func (r *InfluxRunner) checkHealth(ctx context.Context) (*interface{}, error) {
+	// TODO call https://github.com/influxdata/influxdb-client-go/blob/master/client.go#L32
+	return nil, fmt.Errorf("not implemented yet!")
```

Comment:
looks like we need to update latest from master for this... @srclosson how did you do this before?